examples/webcomponents/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv='Content-type' content='text/html; charset=utf-8'>
+    <title>Basic Example with WebComponents</title>
+    <link rel="stylesheet" href="../shared/css/base.css" />
+  </head>
+  <body>
+    <h1>Basic Example with WebComponents</h1>
+    <div id="container">
+      <p>
+        To install React, follow the instructions on
+        <a href="http://www.github.com/facebook/react/">GitHub</a>.
+      </p>
+      <p>
+        If you can see this, React is not working right.
+        If you checked out the source from GitHub make sure to run <code>grunt</code>.
+      </p>
+    </div>
+    <br /><br />
+    <h4>Example Details</h4>
+    <p>
+        This example demonstrates WebComponent/ReactComponent interoperability
+        by rendering a ReactComponent, which renders a WebComponent, which renders
+        another ReactComponent in the WebComponent's shadow DOM.
+    <p>
+    <p>
+      Learn more about React at
+      <a href="http://facebook.github.io/react" target="_blank">facebook.github.io/react</a>.
+    </p>
+    <script src="../shared/thirdparty/es5-shim.min.js"></script>
+    <script src="../shared/thirdparty/es5-sham.min.js"></script>
+    <script src="../shared/thirdparty/console-polyfill.js"></script>
+    <script src="../shared/thirdparty/webcomponents.js"></script>
+    <script src="../../build/react.js"></script>
+    <script src="../../build/JSXTransformer.js"></script>
+
+    <!-- Define WebComponent -->
+    <script type="text/jsx">
+      var proto = Object.create(HTMLElement.prototype, {
+        createdCallback: {
+          value: function() {
+            var mountPoint = document.createElement("span");
+            this.createShadowRoot().appendChild(mountPoint);
+
+            var url = "https://www.google.com/search?q="+encodeURIComponent(this.innerHTML)+"&btnI";
```

Comment:
Small nit to match the rest of our code: single quotes and spaces around operators